ci/README.md

```diff
@@ -45,6 +45,12 @@ To build for armv7 for example:
 ./build.py -p armv7 /work/functions.sh build_armv7
```

Comment:
Would you mind adding instructions where the build artefacts are going to be located?

Reply:
done